crawler/regelleistung.py

```diff
@@ -0,0 +1,297 @@
+# SPDX-FileCopyrightText: Steffen Carstensen
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+"""
+https://regelleistung.net/
+"""
+import logging
+import sys
+import warnings
+
+import pandas as pd
+import numpy as np
+from sqlalchemy import create_engine
+from datetime import date, timedelta, datetime
+import sqlalchemy
+
+from .config import db_uri
+
+
+log = logging.getLogger("regelleistung")
+log.setLevel(logging.INFO)
+
+DEFAULT_MAX_DAYS_PAST = 365*2
```

Comment:
wouldn't it make more sense to state "2020-01-01" or something here?

Reply:
Done